docs/docs/how-tos/nebari-aws.md

```diff
@@ -0,0 +1,132 @@
+:::
+
+## Authentication
+
+In order to Nebari make requests against the aws API and create it's infrastructure, an authentication method with the appropriate permissions will be required. The easiest way to do this is using a [IAM user](https://docs.aws.amazon.com/IAM/latest/UserGuide/introduction.html) to allow access over your AWS account and EKS cluster management.
+
+As a [best practice](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#lock-away-credentials), do not use the AWS account root user for any task where it's not required. Instead, create a new IAM user for each person that requires administrator access. Then make those users administrators by placing the users into an "Administrators" user group to which you attach the `AdministratorAccess` managed policy.
+
+If its your first time creating a new IAM user, please refer to [Creating your first IAM admin user and user group](https://docs.aws.amazon.com/IAM/latest/UserGuide/getting-started_create-admin-group.html) for more information. Otherwise, see [Creating an IAM user in your AWS account](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_create.html) for more information on how to create a IAM user.
+
+Follow these steps to set up your access keys and user accounts:
+
+- Go to [IAM in your AWS Console](https://console.aws.amazon.com/iam/home), then **Users**, and **Add Users**;
+- Give the user a name, and tick **Access Key - Programmatic access**
+- Click **Next**
+
+![alt text for screen readers](/img/how-tos-aws-new-iam-user.png "Text to show on mouseover")
+
+- Select **Attach existing policies directly**, then select `AdministratorAccess` from the list of policies. For more information, please refer to [Policies and permissions in IAM](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies.html);
+- Then proceed with the new user creation setup.
+
+Upon generation, the IAM role will provide a public **Access Key ID** and **Secret Access Key**. Download this file for reference later.
+
+:::warning Warning
+The **Secret Access Key** provides access to your AWS user account. It should be treated like any other secret credentials. Specifically, it should never be checked into source control.
+:::
+
+By default, Nebari will try to use the credentials associated with the current AWS infrastructure/environment for authentication. Please keep in mind that Nebari will only use these credentials to create the first roles and stricter permissions for Nebari's internal components. Refer to [Conceptual guides] for more information on how Nebari's components are secured.
```

Comment:
> By default, Nebari will try to use the credentials associated with the current AWS infrastructure/environment for authentication.

As opposed to what other credentials?

> Please keep in mind that Nebari will only use these credentials to create the first roles and stricter permissions for Nebari's internal components.

Is this meant to emphasize that Nebari is only going to use the *administrator level* credentials to create roles with lesser permissions, and then those lower-permission roles will be used to operate Nebari's internal components?

----

There's an incomplete link here on `[Conceptual guides]` -- looks like the page just may not exist yet?

Will there also be a conceptual guide as to how Nebari's internal components are laid out, and not just how they're secured?

Reply:
@bskinn let me know what you think, any ideas on how I can re-write that part?

Reply:
I think this is ok - may be worth linking to the AWS (and equivalent for other providers) as to where AWS looks for credentials: https://docs.aws.amazon.com/sdkref/latest/guide/file-location.html